exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,13 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr.shift(); // complete this statement
```

Comment:
While this returns the first element, "shift" also removes that element from the array. Since an array is a javascript object, it is passed by reference (that means the function can change what you pass to it), meaning if you called this function, it would remove the first element from the function, which may be quite an unexpected side-effect.

```javascript
function first(arr) {
    return arr[0]
}
```

May be better here.